backend/src/apiserver/client_manager.go

```diff
@@ -247,7 +247,8 @@ func initMysql(driverName string, initConnectionTimeout time.Duration) string {
 
 	// Create database if not exist
 	operation = func() error {
-		_, err = db.Exec(fmt.Sprintf("CREATE DATABASE IF NOT EXISTS %s", dbName))
```

Comment:
Just get the name once and use it below:

dbName := getStringConfig(mysqlDBName)

No need for the default, we can keep default in DBConfig struct in the JSON. That way, defaults are all in one place, and not spread out in code.